<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cryptographic</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images.ico</normaloff>images.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame1">
       <property name="styleSheet">
        <string notr="true">color: rgb(160, 160, 160);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="teSrcFile">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>160</y>
          <width>311</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Kiểu Mã Hóa/ Giải Mã</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>File Cần Mã Hóa/Giải Mã</string>
        </property>
       </widget>
       <widget class="QComboBox" name="EncryptionType">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>100</y>
          <width>311</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <item>
         <property name="text">
          <string>Mặc Định</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mật mã Ceasar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mật mã RC4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mật mã rail fence</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mật mã Vigenère</string>
         </property>
        </item>
       </widget>
       <widget class="QToolButton" name="OpenFileSrc">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>160</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,0,0);</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>200</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,0,0);</string>
        </property>
        <property name="text">
         <string>Kiểu Lưu Trữ</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>250</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,0,0);</string>
        </property>
        <property name="text">
         <string>Chọn Folder</string>
        </property>
       </widget>
       <widget class="QWidget" name="wKieuluu" native="true">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>190</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
        <widget class="QRadioButton" name="rdoldFile">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0,0,0);</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>Ghì Đè</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="rdnewFile">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>10</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0,0,0);</string>
         </property>
         <property name="text">
          <string>File Mới</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>241</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>35</pointsize>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);
</string>
        </property>
        <property name="text">
         <string>Cryptography</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btOk">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>320</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,0,0);</string>
        </property>
        <property name="text">
         <string>Mã Hóa</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btReset">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>320</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,0,0);</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
       <widget class="QToolButton" name="OpenFileSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>250</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,0,0);</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="textSaveFile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>250</y>
          <width>311</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btGM">
        <property name="geometry">
         <rect>
          <x>284</x>
          <y>320</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 0, 0)</string>
        </property>
        <property name="text">
         <string>Giải Mã</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Err_name_filesrc">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>140</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_Err_name_filedes">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>230</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QProgressBar" name="Bar">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>370</y>
          <width>471</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255);
stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
